src/mam4xx/mo_drydep.hpp

```diff
@@ -25,10 +26,111 @@ constexpr Real rair = 287.04;
 constexpr Real grav = 9.81;
 constexpr Real karman = 0.4;   // from shr_const_mod.F90
 constexpr Real tmelt = 273.15; // from shr_const_mod.F90 via physconst.F90
-
+constexpr Real r2d = 180.0 / haero::Constants::pi; // degrees to radians
```

Comment:
This multiplicative factor converts radians to degrees, right? The variable name seems correct, but the comment is not.

Reply:
Yes, it should convert degrees to radians. It was a copy/paste error from [here]( https://github.com/eagles-project/e3sm_mam4_refactor/blob/599e772e34a0a32dacadc40688fd091e1f532d76/components/eam/src/chemistry/utils/mo_constants.F90#L32).

Reply:
But multiplying by 180.0 / pi converts radians to degrees, doesn't it?

2*pi * 180/pi = 2 * 180 = 360

Reply:
Yes. It should be radians to degrees.

Reply:
It is fixed in `mam4xx`.